_layouts/no-header-no-footer.html

```diff
@@ -22,7 +22,13 @@
   <!-- End of HubSpot Embed Code -->
 
   {% if site.jekyll_env == "production" %}
-  <script src="https://cdn.userway.org/widget.js" data-account="RCwaT3oWqd"></script>
+    {% if page.url == "/christmas-story/" %}
+    {% endif %}
+    {% elseif page.url == "/christmas-story" %}
+    {% endif %}
+    {% else %}
+      <script src="https://cdn.userway.org/widget.js" data-account="RCwaT3oWqd"></script>
+    {% endif %}
```

Comment:
This logic looks like it'll work but I wonder if we could do this instead

```
{% if page.url != "/christmas-story/" and page.url != "/christmas-story" %}
  <script src="https://cdn.userway.org/widget.js" data-account="RCwaT3oWqd"></script>
{% endif %}
```

Reply:
Yeah that's definitely better